niworkflows/reports/core.py

```diff
@@ -214,13 +216,13 @@ class Report(object):
 
     >>> robj = Report(testdir / 'work' / 'reportlets', testdir / 'out',
     ...               'madeoutuuid', subject_id='01', packagename='fmriprep')
-    >>> robj.layout.get(subject='01', desc='reconall')
-    [<BIDSFile filename='anat/sub-01_desc-reconall_T1w.svg'>]
+    >>> robj.layout.get(subject='01', desc='reconall')  # doctest: +ELLIPSIS
+    [<BIDSFile filename='.../anat/sub-01_desc-reconall_T1w.svg'>]
 
     >>> robj.generate_report()
     0
     >>> len((testdir / 'out' / 'fmriprep' / 'sub-01.html').read_text())
-    19352
+    19369
```

Comment:
Unclear what's producing changes in the length of the file.

Reply:
Longer paths under each reportlet?